src/test/cli/rbd/help.t

```diff
@@ -420,6 +423,55 @@
     -p [ --pool ] arg    pool name
     --group arg          group name
   
+  rbd help group image add
+  usage: rbd group image add [--pool <pool>] [--group <group>] 
+                             <group-spec> <image-spec> 
+  
+  Add an image to a consistency group.
+  
+  Positional arguments
+    <group-spec>         group specification
+                         (example: [<pool-name>/]<group-name>)
+    <image-spec>         image specification
+                         (example: [<pool-name>/]<image-name>)
+  
+  Optional arguments
+    -p [ --pool ] arg    pool name
+    --group arg          group name
```

Comment:
A question about another command, but, I guess, relevant.
List images in a consistency groups command only needs group and pool names: would you prefer option 1 or option 2.

Option 1:
group image list
--pool pool name
--group group name

Option 2:
group image list
--group-pool pool name
--group group name